### PR TITLE
Detection of flags

### PR DESCRIPTION
**Pull request by siddharth9820**
Blackens entire background except flags.
1) HSV code for white is giving erroneous results. Hence the thresholded flags are of different colours.

### CODE REVIEW

File: Flag_Detection.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ 
+ }
+return hsv;
+}
+
+int main()
+{
+Mat image,t,p;
+image=imread("test_flags.jpg");
+namedWindow("Display Original", CV_WINDOW_AUTOSIZE);
```
Comment (krishna95) [inline Flag_Detection.cpp]: Please indent the code properly

Comment (siddharth9820) [inline Flag_Detection.cpp]: You mean indent?

Comment (krishna95) [inline Flag_Detection.cpp]: Yes, sorry my bad

**Comment by krishna95** [comment]
@aranyadan Please review

**Comment by aranyadan** [comment]
Remove temporary files (file names ending with ~)

**Comment by siddharth9820** [comment]
@aranyadan The correct commit is #31. Sorry for the confusion.

**Pull request closed without merging**